docs/userguide.rst

```diff
@@ -166,6 +166,12 @@ This can be used if you have control over the global site-packages directory,
 and you want to depend on the packages there. If you want isolation from the
 global system, do not use this flag.
 
+If you need to change this option after creating a virtual environment, you can 
+add (to turn off) or remove (to turn on) the file ``no-global-site-packages.txt`` 
+from ``lib/python2.7/`` or equivalent in the environments directory. If you have 
```

Comment:
I would rather not have mention here of higher-level abstractions/tools. Let's just remove the last sentence and it's good to go (that should be part of ``virtualenvwrapper`` documentation, not here).

Reply:
done..